external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -39,13 +39,16 @@ import org.apache.spark.tags.DockerTest
  * while Spark QA test run.
  *
  * The following would be the steps to test this
- * 1. Pull oracle 11g image - docker pull wnameless/oracle-xe-11g
- * 2. Start docker - sudo service docker start
- * 3. Download oracle 11g driver jar and put it in maven local repo:
+ * 1. Build Oracle database in Docker, please refer below link about how to.
+ *    @see https://github.com/oracle/docker-images/blob/master/OracleDatabase/SingleInstance/README.md
```

Comment:
The line is too long, and the `@see` tag doesn't work here I think. Put "See" on the previous line and unindent to see if that gets it to under 100 characters